### PR TITLE
Release eds-core-react@0.6.2

### PR DESCRIPTION
**Pull request by mimarz**
### Fixed 🐛

- Fixed `onClose` handler not being called when `MenuItem` was used inside `MenuSection` ([#546](https://github.com/equinor/design-system/issues/546))
- Fixed focus showing when clicking on `MenuItem` ([#544](https://github.com/equinor/design-system/issues/544))
- Fixed an issue where `Divider` did not stretch to full width ([#608](https://github.com/equinor/design-system/issues/608))
- Fixed `Head` styling when `Table` is set to `width: 100%` ([#610](https://github.com/equinor/design-system/issues/610))

### Changed

- Added outside click support for closing `Menu`. Outside clicks will now call the `onClose` handler function. ([#548](https://github.com/equinor/design-system/issues/548))
- Added `data` property to `Icon` component to easily compose icon to be rendered. ([#584](https://github.com/equinor/design-system/issues/584))
  - See `Icon` [README](https://github.com/equinor/design-system/tree/develop/libraries/core-react/src/Icon) for more information

### CODE REVIEW

**Review by pomfrida — APPROVED** [review]
👍

File: libraries/core-react/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,33 @@ All notable changes to this project will be documented in this file.
...
+<Icon data={save} />
+
+// Custom icon
+<Icon data={{
+  prefix: 'custom',
+  height: '24',
+  width: '24',
+  svgPathData: 'M12 16.067l4.947 3.6-1.894-5.814L20 10.334h-6.067l-1.933-6-1.933 6H4l4.947 3.52-1.894 5.814 4.947-3.6z',
+}}>
+```
```
Comment (vnys) [inline libraries/core-react/CHANGELOG.md:33]: Don’t include code examples in the changelog, better to link to the storybook

**Review by vnys — CHANGES_REQUESTED** [review]
See comment about code example

File: libraries/core-react/CHANGELOG.md (hunk 1 above)
Comment (mimarz) [reply libraries/core-react/CHANGELOG.md:33]: @vnys Storybook does not show import statements unfortunately, it only shows the content on the render function you export 🤔

Comment (vnys) [reply libraries/core-react/CHANGELOG.md:34]: A README file somewhere then?

Comment (mimarz) [reply libraries/core-react/CHANGELOG.md]: @vnys Good suggestion, linked to README file for `Icon` component instead

**Pull request merged**